vmware_aria_operations_integration_sdk/mp_init.py

```diff
@@ -427,7 +427,7 @@ def build_project_structure(
         # create template requirements.txt
         requirements_file = os.path.join(path, "adapter_requirements.txt")
         with open(requirements_file, "w") as requirements:
-            requirements.write("psutil==5.9.0\n")
+            requirements.write("psutil==5.9.4\n")
```

Comment:
I had noticed this was an issue recently but didn't look into it at the time. Do you know what caused the older version to stop working? (Was it from when we updated the base OS image?)